.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -128,6 +131,8 @@ jobs:
     - name: check-sycl
       if: always()
       run: |
+        # TODO consider moving this to Dockerfile
+        export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
```

Comment:
According to https://intel.github.io/llvm-docs/GetStartedGuide.html#build-dpc-toolchain-with-support-for-nvidia-cuda, this should not be required. I suggest filing an issue to investigate what is wrong here. Maybe NVIDIA docker image is not configured correctly.

I have the same question regarding `--hip-amd-arch` option. I don't see it listed as required option in the GetStartedGuide. Should we add it there?

Reply:
> Maybe NVIDIA docker image is not configured correctly.

That is actually the case

> I have the same question regarding --hip-amd-arch option. I don't see it listed as required option in the GetStartedGuide. Should we add it there?

It is mentioned here: https://intel.github.io/llvm-docs/GetStartedGuide.html#run-in-tree-lit-tests
The option is not required to build the compiler, only to test it.